core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -243,19 +243,19 @@ final case class DefaultKubernetesLabels(googleProject: GoogleProject,
 
 sealed abstract class ErrorAction
 object ErrorAction {
-  case object CreateGalaxyApp extends ErrorAction {
+  case object CreateApp extends ErrorAction {
```

Comment:
think we need to update `toString` as well

Reply:
I didn't want to update the toString because this field is persisted to the database and decoded. It's a fairly high-risk change (though not necessarily complex), so I split it into this ticket https://broadworkbench.atlassian.net/browse/IA-2580